packages/framework/presence/src/presence.ts

```diff
@@ -60,8 +76,18 @@ export interface ISessionClient<
 	 *
 	 * @remarks
 	 * Connection id will change on reconnect.
+	 *
+	 * If {@link ISessionClient.status} is {@link (SessionClientStatus:variable).Disconnected}, this will represent the last known connection id.
+	 */
+	connectionId(): ClientConnectionId;
+
+	/**
+	 * Get status of session client.
+	 *
+	 * @returns Status of session client.
+	 *
 	 */
-	currentConnectionId(): ClientConnectionId;
+	status(): SessionClientStatus;
```

Comment:
I think making this a `get` property will be easier to use. It seems unlikely that the "status" function ever needs arguments, so a getter makes it dynamic but the usage is a little cleaner. Not critical.